docs/sources/write/shortcodes/index.md

```diff
@@ -541,6 +541,21 @@ Including the original image dimensions as the 'width' and 'height' properties i
 These values are _only_ used for determining the image aspect ratio and don't equate to the final displayed size.
 {{< /admonition >}}
 
+### Center an image example
+
+You can center an image using the `figure` shortcode by adding the following properties:
+
+- Add the `class="w-100p"` property.
+- Add the `link-class="w-fit mx-auto d-flex flex-direction-column"` property.
+- Add the `max-width="WIDTHpx"` property, replacing `WIDTH` with the value you want for your image. The value should be lower than the original image's width.
+- Optionally, you can add the `width` and `height` properties with the original image values, without pixels or percentages. For example, for a 800x600 image, you can add `width="800" height="600"`.
+
```

Comment:
@jdbaldry I think this set of instructions should be enough to help someone center an image, but Nate also shared with me a longer explanation for each property, which I think could be helpful if someone wants to understand what exactly each property does...

I was thinking of adding it here using the `collapse` shortcode, do you think that's an ok use for it? 🤔